test/pipelines/pipelines-it-installation.yml

```diff
@@ -16,11 +16,7 @@ jobs:
 
```

Comment:
better to remove the `python36` in job name

Reply:
updated